Assets/HoloToolkit/Utilities/Scripts/Throwing.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.XR;
```

Comment:
Not properly wrapped to support VR namespace

Reply:
I'm confused by this.  It seems in order to 'correctly' wrap it, I would need to add a ton of #if-ed code, that is only destined to go out of date, this won't be in a version or MRTK that is intended for <2017.1.